hoomd/md/force.py

```diff
@@ -64,6 +64,10 @@ class Force(Compute):
         for `isinstance` or `issubclass` checks.
     """
 
+    # Which method to call on pybind11 exposed SystemDefinition to get correct
+    # numbers (e.g. getN) for the force. Must be overwritten by other classes.
+    _cpp_data_method = ""
```

Comment:
This "fix" introduces a bug. All `ForceCompute` objects compute forces on each particle and store it in a `N_particles` sized array.

Reply:
Good point. For some reason, my mind was thinking that the forces must be per-group, but of course these apply to individual particles ultimately. I will revert this "fix" as you said.